### DISCUSSION TITLE
Welcome to Llama 2 Everywhere Discussions!

### DISCUSSION THREAD

**Discussion by trholding** [trholding/llama2.c#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by majehuang** [comment]
How should I train it, does it support fine-tuning?
